mmv1/products/logging/LogView.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: LogView
+base_url: '{{parent}}/locations/{{location}}/buckets/{{bucket}}/views'
+create_url: '{{parent}}/locations/{{location}}/buckets/{{bucket}}/views?viewId={{name}}'
+self_link: '{{parent}}/locations/{{location}}/buckets/{{bucket}}/views/{{name}}'
+update_verb: :PATCH
+update_mask: true
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    "Official Documentation": "https://cloud.google.com/logging/docs/apis"
+  api: "https://cloud.google.com/logging/docs/reference/v2/rest/v2/projects.locations.buckets.views"
+description: "Describes a view over log entries in a bucket."
+parameters:
+  - !ruby/object:Api::Type::String
+    name: parent
```

Comment:
Was computed before and had `compareSelfLinkOrResourceName` diffsuppressfunc

Reply:
This is O+C currently because the DCL fills `parent` and `location` using the `bucket` field if they are unspecified. I'm working on replicating this behavior in MMv1.